app/controllers/cases_controller.rb

```diff
@@ -41,6 +41,7 @@ class CasesController < ApplicationController
                   :approve_response_interstitial,
                   :assign_to_new_team,
                   :close,
+                  :closure_outcomes,
```

Comment:
This and the other actions here should be moved to `set_case`, and @case should be decorated as a separate step.